internal/shortener/repository/repository.go

```diff
@@ -132,29 +172,184 @@ func (s *StoreFile) GetShortener(req *GetShortenerRequest) (*GetShortenerRespons
 	}, nil
 }
 
-func (s *StoreFile) SetShortener(req *SetShortenerRequest) {
+func (s *StoreFile) SetShortener(ctx context.Context, req *SetShortenerRequest) (*SetShortenerResponse, error) {
```

Comment:
```suggestion
func (s *StoreFile) SetShortener(_ context.Context, req *SetShortenerRequest) (*SetShortenerResponse, error) {
```
и линтер не будет ругаться.